packages/harmony/src/components/select/SelectPopupKeyHandler.tsx

```diff
@@ -1,13 +1,13 @@
 import { RefObject, useState, ReactNode, useEffect } from 'react'
 
-import { FilterButtonOption } from '../FilterButton/types'
+import { SelectOption } from './Select.types'
```

Comment:
interesting. Are we moving towards `X.types.ts` files over `X/types.ts` files in the future?

Reply:
Idk I just had multiple components in this single directory and wanted to keep the types separate, but maybe I should put them in their own directories